### PR TITLE
Fix embedwast.py for out-of-tree building

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Comment by sbc100** [comment]
@alexcrichton, does this seems reasonable?

**Comment by alexcrichton** [comment]
👍 

Sorry about that!

File: scripts/embedwast.py [hunk 1]
```diff
@@ -1,31 +1,45 @@
...
 for c in wast:
-    output += str(ord(c)) + ', '
-output += '0'
-output += "\n};\n"
-output += "const char* IntrinsicsModuleWast = theModule;\n"
-output += "}\n"
-open(sys.argv[1], 'w').write(output)
+  output += str(ord(c)) + ', '
+
+output += '''0
```
Comment (jgravelle-google) [inline scripts/embedwast.py]: nit: why is this triple-quoted string using `'''`, when the other one uses `"""`?
Is there a standard for which to use? Either way these should be the same.

Comment (kripken) [reply scripts/embedwast.py]: Personally I think `'` is nicer as it requires fewer keypresses on most keyboards.

**Pull request merged**